integrations/RollBar/browser.js

```diff
@@ -0,0 +1,496 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+
+import { isObject } from "../../utils/utils";
+import logger from "../../utils/logUtil";
+import { NAME } from "./constants";
+
+class RollBar {
+  constructor(config) {
+    this.name = NAME;
+    this.accessToken = config.accessToken;
+    this.captureUncaughtException = config.captureUncaughtException;
+    this.captureUnhandledRejections = config.captureUnhandledRejections;
+    this.guessUncaughtFrames = config.guessUncaughtFrames;
+    this.codeVersion = config.codeVersion;
+    this.ignoredMessages = config.ignoredMessages;
+    this.environment = config.environment;
+    this.sourceMapEnabled = config.sourceMapEnabled;
+  }
+
+  init() {
+    logger.debug("===in init RollBar===");
+    var _rollbarConfig = {
+      accessToken: this.accessToken,
+      captureUncaught: this.captureUncaughtException,
+      captureUnhandledRejections: this.captureUnhandledRejections,
+      payload: {
+        environment: this.environment,
+        client: {
+          javascript: {
+            code_version: this.codeVersion,
+            source_map_enabled: this.sourceMapEnabled,
+            guess_uncaught_frames: this.guessUncaughtFrames,
+          },
+        },
+      },
+    };
+    var msg = this.ignoredMessages;
```

Comment:
Please verify is nothing is enabled, an empty string is coming. and if empty string is sent, rollbar behaves properly.

Reply:
```
  "ignoredMessages": [
              {
                "eventName": ""
              }
            ]
```
It will come as this, you need to form the actual list of messages from here. Also the variable name should change from eventName to `singleIgnoredMessage`

Reply:
fixed the change